rs-cart-api/src/cart/cart.controller.ts

```diff
@@ -0,0 +1,112 @@
+import { Controller, Get, Delete, Put, Body, Req, Post, HttpStatus, UseGuards } from '@nestjs/common';
+import { BasicAuthGuard } from 'src/auth';
+
+// import { BasicAuthGuard, JwtAuthGuard } from '../auth';
+import { OrderService } from '../order';
+import { AppRequest, getUserIdFromRequest } from '../shared';
+
+import { calculateCartTotal } from './models-rules';
+import { CartService } from './services';
+
+@Controller('api/profile/cart')
+export class CartController {
+  constructor(
+    private cartService: CartService,
+    private orderService: OrderService
+  ) { }
+
+  // @UseGuards(JwtAuthGuard)
+  // @UseGuards(BasicAuthGuard)
+  @Get()
+  async findUserCart(@Req() req: AppRequest) {
+    try {
+      const cart = await this.cartService.findOrCreateByUserId('123e4567-e89b-12d3-a456-426614174000');
```

Comment:
It looks like it's the wrong service method, it should be `this.cartService.getCart()`